pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -58,11 +64,28 @@ Examples:
 				newBuild = buildutil.GenerateBuildFromBuild(build)
 			}
 
+			var wg sync.WaitGroup
+			if streamLogs {
+				rd, err := client.Get().Namespace(namespace).Prefix("redirect").Resource("buildLogs").Name(buildName).Stream()
```

Comment:
> i'm trying to understand why you're streaming the logs of buildName(really a buildconfig name) and not newBuild.Name

Good catch, I am probably streaming buildConfig logs. Is there such a things as buildConfig logs?

> if i invoke "osc start-build foo --follow" twice simultaneously in two consoles, will I see two different logs or the same log twice?

What happens when you try to start the same build twice?